Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -243,4 +246,11 @@ def Runner(env):
             # Tested same FDs on QEMU 6 and 7, not observing the same.
             ret = 0
 
+        if os.name == 'nt' and qemu_version[0] >= '8':
+            # Restore the console mode for Windows on QEMU v8+.
+            std_handle.SetConsoleMode(console_mode)
+        elif os.name != 'nt':

```

Comment:
This will run whenever the os is not nt. Is that what you want? or only when qemu verison is > 8?